### DISCUSSION TITLE
Explain terms used in constraints

### DISCUSSION THREAD

**Discussion by wdfinch** [yarnpkg/berry#2698] [Q&A] [answered]
I'm struggling to understand what some of the terms used in constraints mean. Unfortunately, the docs do not provide definitions for everything and it's not obvious (at least to me) what some things mean. I would really appreciate if someone could provide a brief definition for each of these: 

1. `WorkspaceCwd`
2. `WorkspaceIdent`

**Comment by merceyz** [comment, marked as answer]
`WorkspaceCwd` is the relative path to a workspace (`./packages/yarnpkg-core`) and `WorkspaceIdent` is the name of a workspace (`@yarnpkg/core`)

**Reply by wdfinch** [reply]
Thanks!
